db/migrate/20200221153634_drop_moments_mood_column.rb

```diff
@@ -0,0 +1,5 @@
+class DropMomentsMoodColumn < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :moments, :mood
```

Comment:
Likely won't be a big deal, but [dropping a column can cause exceptions during deployment](https://github.com/ankane/strong_migrations#removing-a-column) until the app is rebooted. If we want to avoid that, it's recommended that we queue up two migrations and deploy them separately.

Reply:
Ooh great point!

Reply:
Ahh I see. Thanks for the link.